legal-api/src/legal_api/reports/business_document.py

```diff
@@ -297,25 +297,27 @@ def _set_directors(self, legal_entity: dict):
 
     def _set_parties(self, legal_entity: dict):
         """Set the parties of the business (all parties)."""
-        party_json = get_parties(self._legal_entity.identifier).json["parties"]
+        party_json = get_parties(self._business.identifier).json["parties"]
         for party in party_json:
             if party.get("mailingAddress"):
                 party["mailingAddress"] = BusinessDocument._format_address(party["mailingAddress"])
             if party.get("deliveryAddress"):
                 party["deliveryAddress"] = BusinessDocument._format_address(party["deliveryAddress"])
+            if not party.get("officer", {}).get("partyType") and not party.get("officer", {}).get("lastName"):
+                party["officer"]["partyType"] = "organization"
```

Comment:
let's remove this and make the following update to colin_entity model.

![image](https://github.com/bcgov/lear/assets/6685223/92dd4bbe-4d12-426a-8334-13d44aa31d33)